src/common/components/player.h

```diff
@@ -16,10 +16,14 @@
 */
 #pragma once
 
+#include <entt/entt.hpp>
+
 namespace cqsp {
 namespace common {
 namespace components {
-class Player {};
+struct Player {
+
+};
 }  // namespace components
```

Comment:
Delete all the changes in this file, unnecessary changes

Reply:
I disagree, its a struct not a class